terraform/terraform_challenge/terraform.tfstate

```diff
@@ -0,0 +1,2365 @@
+{
```

Comment:
Generally, in the real-world, we'd use S3 & DynamoDB for managing state files.  However, this is fine for dev testing.